### PR TITLE
Allow local decorators via params

### PR DESCRIPTION
**Pull request by tmeasday**
for #5781

## What I did

I added support for local decorators via a parameter. This doesn't resolve the issue but at least it provides a palatable alternative.

### CODE REVIEW

File: lib/client-api/src/client_api.js [hunk 1]
```diff
@@ -203,7 +203,12 @@ export default class ClientApi {
         },
         {
           applyDecorators: this._decorateStory,
-          getDecorators: () => [...localDecorators, ..._globalDecorators, withSubscriptionTracking],
+          getDecorators: () => [
+            ...(allParam.decorators || []),
```
Comment (shilman) [inline lib/client-api/src/client_api.js]: @tmeasday not part of the change per se, but why is this called `allParam`?

**Review by shilman — APPROVED** [review]

**Comment by tmeasday** [comment]
🤷‍♀️

File: lib/client-api/src/client_api.js (hunk 1 above)
Comment (tmeasday) [reply lib/client-api/src/client_api.js]: @shilman I guess it's a combination of all parameters (local global etc) in a similar way?

Comment (tmeasday) [reply lib/client-api/src/client_api.js]: This is why I meant that `addDecorator` could just be redefined to `addDecorator(d) => addParameters({ decorators: [d] })`
